source/ui/ProjectCreationScreen.qml

```diff
@@ -37,156 +45,212 @@ Item {
     property var baseExtensionsChecked: [];
     property var extensionsChecked: [];
 
-    //create the project
-    Button {
-        id: button
-        anchors {
-            top: optionSelector.bottom
-            horizontalCenter: parent.horizontalCenter
+    /////////////////
+    // Project Name
+    /////////////////
 
-            topMargin: marginHeight
-            bottomMargin: marginHeight
-            leftMargin: marginWidth
-            rightMargin: marginWidth
-        }
-        text: "create Project"
-        onClicked: {
-            tab.title = textInputName.text;
-            root.buttonClicked(memorySizeSelector.getCurrentValue(),
-              architectureSelector.currentText, optionSelector.currentText, parserSelector.currentText);
+    ProjectCreationSection {
+      id: projectName
+      anchors.topMargin: Theme.createProject.marginTop
+      TextField {
```

Comment:
If you want to avoid an empty project name, you could add something like this:
```
onAccepted: {
  if (text === "") {
    button.enabled = false;
  } else {
    button.enabled = true;
  }
}
```